x/stakeibc/keeper/host_zone.go

```diff
@@ -92,6 +93,32 @@ func (k Keeper) GetAllHostZone(ctx sdk.Context) (list []types.HostZone) {
 	return
 }
 
+func (k Keeper) AddValidatorToHostZone(ctx sdk.Context, chainId string, val types.Validator) (success bool) {
+	hostZone, found := k.GetHostZone(ctx, chainId)
+	if !found {
+		k.Logger(ctx).Error(fmt.Sprintf("HostZone not found %s", chainId))
+		return false
+	}
+	hostZone.Validators = append(hostZone.Validators, &val)
```

Comment:
should we check whether it already exists and handle that accordingly, maybe "editValidator" if we try to add some with address equal to one that already exists?